Damage.cs

```diff
@@ -6,7 +6,6 @@ namespace PhysicalDamage.Core
 {
     abstract class Damage
     {
-        #region DamageType
         protected EDamageTypes damageType;
```

Comment:
We can shorten this code using auto-properties.

`protected EDamageTypes DamageType { get; set; }`
or if you want it to be read-only and only able to be set within the class or derived classes
`protected EDamageTypes DamageType { get; private set; }`